FaunaDB.Client/Query/Language.Read.cs

```diff
@@ -72,5 +72,32 @@ public partial struct Language
         public static Expr Reduce(Func<Expr, Expr, Expr> lambda, Expr initial, Expr collection) =>
             Reduce(Lambda(lambda), initial, collection);
 
+        /// <summary>
+        /// Returns the number of items that exist in the array or set
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/count">FaunaDB Count Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Count(params Expr[] terms) =>
```

Comment:
I don't think vargars would be a good idea, calling `Count(1, 2, 3)` is nice, but it would allow `Count(Match(...), Match(...))` which is not allowed

https://github.com/fauna/faunadb-jvm/blob/master/faunadb-java/src/main/java/com/faunadb/client/query/Language.java#L6627